### ISSUE TITLE
Rewriting the introduction is 'What is the shell?'

### ISSUE THREAD

**Issue by enerammer** [LibraryCarpentry/lc-shell#158]
I think some key points about the shell is not clearly explained in the introduction to the lesson 'What is the shell?'. It could be a good idea to look at the 'background' section here http://swcarpentry.github.io/shell-novice/01-intro/index.html. It is very clearly explained what a shell is.

**Comment by lizlou** [comment]
I agree, having read the 'background' section I think it would help the participants of the course have a better understanding, especially if they aren't confident in their knowledge. At the moment with most of the training being conducted online, perhaps having some of the information on the prerequisites would save some time?

**Comment by kmacvaug** [comment]
I also concur that drawing on the SW shell introduction would be useful. The Library introduction is heavy with text and I prefer the code examples on ([ http://swcarpentry.github.io/shell-novice/01-intro/index.html. ], as well as the example user profile (could be switched to a librarian working on the ebook metadata project).

**Comment by CSGeer** [comment]
To add to this, I would suggest updating the "Key Points" at the bottom of the lesson to include the following points from the SW lesson linked above:

- The shell’s main advantages are its high action-to-keystroke ratio, its support for automating repetitive tasks, and its capacity to access networked machines.
- The shell’s main disadvantages are its primarily textual nature and how cryptic its commands and operation can be.

I particularly like that it points out the disadvantages of using Shell. 

It might also be worthwhile to use a version of this paragraph from the "Working with files and directories" episode in the "What is shell?" episode:

"As well as navigating directories, we can interact with files on the command line: we can read them, open them, run them, and even edit them. In fact, there’s really no limit to what we can do in the shell, but even experienced shell users still switch to graphical user interfaces (GUIs) for many tasks, such as editing formatted text documents (Word or OpenOffice), browsing the web, editing images, etc. But if we wanted to make the same crop on hundreds of images, say, the pages of a scanned book, then we could automate that cropping work by using shell commands."
